### PR TITLE
Fix errors in README

### PR DESCRIPTION
**Pull request by jisungk**
- add information about updating the connection string information
- update keyword to correct one: `!odbot`
- fix bad links and style errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,16 +35,16 @@ After logging into the Azure Portal, follow these steps to register the sample a
     4. Click **Create** to create the app.
 3. After the app has been created successfully, select the app from the list of applications. It should be at the bottom of the list.
 4. Copy the **Application ID** for the app you registered and paste it into two places:
-    * In the Web.config file on the line: `<add key="ida:ClientId" value="[ClientId]" />`
-    * In the run.csx file on the line: `private const string idaClientId = "[ClientId]";`
+    * In the [`Web.config`](https://github.com/jisungk/onedrive-data-robot-azure-function/blob/jisungk-edits/OneDriveDataRobot/Web.config) file on the line: `<add key="ida:ClientId" value="[ClientId]" />`
```
Comment (rgregg) [inline README.md]: Can you make these links relative links so they don't break when this code is integrated into the original project?

File: README.md [hunk 2]
```diff
@@ -62,11 +62,11 @@ After the required components have been provisioned, click on the **App Services
 
 To create a new Azure Function application and setup a function for this project:
 
-1. Click the `+` next to **Functions**
+1. Click the `+` next to **Functions**.
 2. Select the **Webhook + API** scenario, choose **CSharp** as the language, and then **Create this function**.
-3. Copy the code from [run.csx][] and paste it into the code editor and then click **Save**
-4. On the right side, select **View files** to expand the files that make up this function
-5. Click **Upload** and then navigate to the `project.json` file in the AzureFunction folder and upload it. This file configures the dependencies for the Azure Function, and will add the Azure authentication library and the Microsoft Graph SDK to the function project.
+3. Copy the code from [`run.csx`](https://github.com/OneDrive/onedrive-data-robot-azure-function/blob/master/AzureFunction/run.csx) and paste it into the code editor and then click **Save**.
```
Comment (rgregg) [inline README.md]: Another place for a relative link.

File: README.md [hunk 3]
```diff
@@ -87,8 +87,12 @@ To create a new Azure Function application and setup a function for this project
      * Leave the other parameters with their default values, and then click **Save**.
 7. Click back on the function name in the left navigation column to bring up the code editor.
 8. Click **Get function URL** and copy the URL for this function. 
-   * Paste this value into the **Web.config** file on the line: `<add key="ida:NotificationUrl" value="[azureFunctionServiceUrl]" />`
-
+   * Paste this value into the `Web.config` file on the line: `<add key="ida:NotificationUrl" value="[azureFunctionServiceUrl]" />`
+10. Navigate to the [Azure Portal](https://ms.portal.azure.com) and go to **Storage Accounts** on the left-hand navigation bar.
```
Comment (rgregg) [inline README.md]: Need to remove the ms. from this URL

**Review by rgregg — CHANGES_REQUESTED** [review]
Just minor tweaks that need to be made before I can merge this in.

**Comment by jisungk** [comment]
I implemented the suggested changes in the latest commit, 7d141c5!

**Review by rgregg — APPROVED** [review]

**Pull request merged**